generators/client/templates/angularjs/src/main/webapp/app/admin/user-management/_user-management-dialog.html

```diff
@@ -35,7 +35,7 @@ <h4 class="modal-title" id="myUserLabel" data-translate="userManagement.home.cre
         <div class="form-group">
             <label class="control-label" data-translate="userManagement.login">Login</label>
             <input type="text" class="form-control" name="login"
-                   ng-model="vm.user.login" ng-required="vm.user.id == null" ng-minlength=1 ng-maxlength=50 ng-pattern="/^[_'.@A-Za-z0-9-]*$/">
```

Comment:
That change is when I converted User Management from NG1 -> NG2.  I can't remember a specific reason, but the login is always required and annotated as `@NotNull` in User.java so it makes sense